FordLabs/peoplemover
# Update react-router-dom from version 5.1.2 to 6.3.0

## Issue
Resolves N/A

## What was done
- [x] Update react-router-dom from version 5.1.2 to 6.3.0
- [x] Updated all route related components to use react-router-dom v6 syntax 

## How to test
1. Test that the app works as it did before.
2. Ensure all unit tests pass.
3. Ensure all cypress tests pass.
4. **We will have to deploy to stage to ensure the login redirects and such still work.**
